### ISSUE TITLE
GCJV assets are listed as R2

### ISSUE THREAD

**Issue by thoufer** [thoufer/mbp-metadatacatalog#20]
GCJV assets are listed as R2 but contact is Steve DeMaso, R4 ... check these are correct

**Comment by Rrau** [comment]
emailed Jennie Duberstein (Sonoran JV coordinator) to ask about Carol's replacement and who might be the new contact for this survey and data.

**Comment by Rrau** [comment]
In the original excel spreadsheet Steve sent back to us, it has region 2 listed as the region of interest.  Migratory Birds/Gulf Coast JV.  My guess is it is because a number of the data assets or monitoring surveys occur in Region 2.  Not sure how you want to deal with that....

**Comment by embuco** [comment]
Yes, just wanted to double check that is where they want to leave it ... might be worth checking with them on the REDH survey too b/c Kathy said the GCJV is holding data/info on that survey.  I'll send her an email to get more info first ... standby!

**Comment by Rrau** [comment]
Nathan, did you get a response back regarding the REDH survey?
